Model/Person/Tagging/Builder.php

```
@@ -150,7 +152,7 @@ public function fromSession(CurrentCustomer $currentCustomer)
             );
             /** @var $person Customer */
             return $person;
-        } catch (\Exception $e) {
+        } catch (Exception $e) {
             return null;
```

The exception is swallowed.